### PR TITLE
Add expandable list of selected items to the top of template variable dropdowns

### PR DESCRIPTION
**Pull request by IntegersOfK**
This pull request adds a feature request #14672 that my organization is very keen to see in an upcoming version of Grafana. I think other people will also enjoy the ability to deselect variables without having to explicitly find them in the list.

~~This pull request also reverses some commented-out CSS, correcting the removed-styling of the scroll-bar introduced with #14274.~~ _(Removed from pull request)_

In an effort to minimize changes to existing user workflow with respect to dropdowns, I made the sub-selections into an expandable list with a `+` icon next to the "Selected (#)" text.

Collapsed (now has a clickable `+` symbol):
selected_collapsed

Expanded (now displays the selected items for easy review & deselection):
selected_expanded

One of my favourite parts of this new little feature is that you are also able to deselect items that exist after the 1000+ truncation limit, which when you have thousands of items selectable can be particularly annoying to find in the list (because they're not there when you scroll).

### CODE REVIEW

File: public/sass/components/_scrollbar.scss [hunk 1]
```diff
@@ -106,78 +106,78 @@
...
-//     color-stop(100%, $scrollbarBackground2)
-//   );
-//   border: 1px solid $scrollbarBorder;
-//   border-top: 1px solid $scrollbarBorder;
-//   border-left: 1px solid $scrollbarBorder;
-// }
+// Scrollbars
 //
+
+::-webkit-scrollbar {
```
Comment (torkelo) [inline public/sass/components/_scrollbar.scss]: sadly the custom scrollbar styles breaks native overlay scrollbars, so we need to figure out a way to add this only if there is no native scrollbars, so please remove this from this PR

Comment (IntegersOfK) [reply public/sass/components/_scrollbar.scss]: I understand, and it doesn't really have anything to do with the feature so it was probably never appropriate to include anyway.

**Comment by IntegersOfK** [comment]
I'm not really clear on why the go test is failing considering I didn't change any go code and I built upon the master branch. Happy to keep digging into it though.

**Comment by torkelo** [comment]
There hasn't been much interest in the feature request, https://github.com/grafana/grafana/issues/14672 

and we are planning to rewrite this dropdown soon to react and would like to limi new features until then. So closing this for now. 

We do really appreciate contributions but we also have to limit the number of features we add to keep the code maintainable. So we try to only add features that have a clear demand from a larger set of uses. So if more request this we will definitely consider it again.

**Pull request closed without merging**